tests/Jellyfin.Common.Tests/Json/JsonOmdbConverterTests.cs

```diff
@@ -0,0 +1,99 @@
+            var seasonRootObject = JsonSerializer.Deserialize<OmdbProvider.RootObject>(Input, _options);
+            Assert.NotNull(seasonRootObject);
+            Assert.Null(seasonRootObject?.Awards);
+            Assert.Null(seasonRootObject?.Episode);
+            Assert.Null(seasonRootObject?.Metascore);
+        }
+
+        [Fact]
+        public void Deserialize_Not_Available_Int_Success()
+        {
+            const string Input = "\"N/A\"";
+
+            var result = JsonSerializer.Deserialize<int?>(Input, _options);
+            Assert.Null(result);
+        }
+
+        [Fact]
+        public void Deserialize_Not_Available_String_Success()
+        {
+            const string Input = "\"N/A\"";
+
+            var result = JsonSerializer.Deserialize<string>(Input, _options);
+            Assert.Null(result);
+        }
+
+        [Fact]
+        public void Deserialize_Normal_String_Success()
+        {
+            const string Expected = "Jellyfin";
+            const string Input = "\"Jellyfin\"";
+
+            var result = JsonSerializer.Deserialize<string>(Input, _options);
+            Assert.Equal(Expected, result);
+        }
+
+        [Fact]
+        public void Deserialize_Null_Success()
+        {
+            const string Input = "null";
+
+            var result = JsonSerializer.Deserialize<string>(Input, _options);
+            Assert.Null(result);
+        }
+
+        [Fact]
+        public void Deserialize_Number_Success()
+        {
+            const int Number = 8;
+            const string Input = "8";
+
+            var result = JsonSerializer.Deserialize<int>(Input, _options);
+            Assert.Equal(Number, result);
+        }
+
+        [Fact]
+        public void Deserialize_Quoted_Number_Success()
+        {
+            const int Number = 8;
+            const string Input = "\"8\"";
+
+            var result = JsonSerializer.Deserialize<int>(Input, _options);
+            Assert.Equal(Number, result);
+        }
+
+        [Fact]
```

Comment:
Opinions on making these `Theory`s instead of `Fact`s?
@crobibero @cvium 
Less descriptive, but also less loc

Reply:
IMO it doesn’t matter because the providers already have PRs for removal from server

Reply:
And those tests will move with them, no?

Reply:
Did move some of the tests to Theorys. Had to be multiple as I couldn't get gernerics to work